server/controllers/group.js

```diff
@@ -0,0 +1,356 @@
+import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
+import sendEmail from './sendEmail';
+import sendSMS from './sendSMS';
+import models from '../models';
+
+const Group = models.Group;
+const User = models.User;
+const Message = models.Message;
+
+dotenv.config();
+
+export default {
+  // Create a group, and automatically add the creator to it
+  create: (req, res) => {
+    const userId = req.body.creatorId;
+    const initialMembers = req.body.initialMembers || [];
+    // Validate input
+    let title = req.body.title || '';
+    let description = req.body.description || '';
+    title = title.trim();
+    description = description.trim();
```

Comment:
how does the above validate your form data? What I can see here is that you can insert an empty string into your database

Reply:
There is a more efficient way to validate your form input

Reply:
You can try to checkout express-validator or do it manually